charactersheet/charactersheet/templates/dm/encounter_sections/player_text_section.tmpl.html

```diff
@@ -34,6 +37,14 @@
       <td data-bind="html: longDescription, click: $parent.editPlayerText"
           class="col-md-12 hidden-sm hidden-xs"
           href="#" data-toggle="modal"></td>
+      <!-- ko if: $parent.shouldShowPushButton -->
+      <td class="text-center">
+        <a data-bind="click: $parent.pushModalToPlayerButtonWasPressed" href="#">
+          <i class="fa fa-paper-plane-o fa-color-hover">
+          </i>
+        </a>&nbsp;
```

Comment:
Found the issue with the random underscore in the send column. It's this `nbsp;`